sass/layout/_sidebar-left.scss

```diff
@@ -46,6 +46,72 @@
         width: 240px;
         background-color: var(--sidebar-header-bg);
     }
+
+    .SidebarChannelNavigator {
+        display: flex;
+        margin: 18px 16px 11px 16px;
+
+        button {
+            font-size: 16px;
+            line-height: 16px;
+            height: 28px;
+            min-width: 28px;
+            vertical-align: middle;
+            border: 0;
+            background: rgba(255, 255, 255, 0.16); // TODO themeing?
```

Comment:
Is this TODO something to get done for this ticket?

Reply:
No, it's a question I have to Andrew about how themes will be handles w.r.t. this widget. The design calls for just an opacity filtered button, which I think is fine, but we'll see once we get to PM/UX reviews.

Reply:
As long as we make sure to clean this up before we're done this feature. I want to avoid leaving TODOs in the code base that'll be forgotten about